docs: improve HMR performance note for Windows users (#31301)
nuxt/nuxt: docs/1.getting-started/2.installation.md

  :summary[Additional notes for an optimal setup:]
  - **Node.js**: Make sure to use an even numbered version (18, 20, etc)
  - **Nuxtr**: Install the community-developed [Nuxtr extension](https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode)

>   - **Windows Users:** For significantly improved HMR performance, we recommend using [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install) with a distribution like Ubuntu. This can reduce reload times to under a second and avoid multiple restarts, providing a smoother development experience.

Reviewer: ```suggestion
  - **WSL**: If you're using Windows, we recommend using [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install) with a distribution like Ubuntu for a better developer performance.
```
